src/menus/menu-divider.jsx

```diff
@@ -1,59 +1,36 @@
 import React from 'react';
-import StylePropable from '../mixins/style-propable';
-import ListDivider from '../lists/list-divider';
-import DefaultRawTheme from '../styles/raw-themes/light-raw-theme';
-import ThemeManager from '../styles/theme-manager';
+import Divider from '../divider';
+import Styles from '../utils/styles';
```

Comment:
I think we should do a whole rewrite with these, I'm still wrapping my head around how styles are actually handled -_-